Geometry/VeryForwardGeometryBuilder/src/DetGeomDesc.cc

```diff
@@ -11,66 +11,113 @@
 
 #include "Geometry/VeryForwardGeometryBuilder/interface/DetGeomDesc.h"
 #include "Geometry/VeryForwardGeometryBuilder/interface/CTPPSDDDNames.h"
+#include "CondFormats/GeometryObjects/interface/PDetGeomDesc.h"
+#include "CondFormats/PPSObjects/interface/CTPPSRPAlignmentCorrectionData.h"
 
+#include "DataFormats/Math/interface/CMSUnits.h"
```

Comment:
I don't think this header is used here. It would be quite confusing if it were being used with the conflicting `GeantUnits.h`.

Reply:
@cvuosalo Yes thanks will remove. At first was using 1. /1_mm, hence the cms units, but then was switched to geant4 units so this header will be removed.